src/server-settings.ts

```diff
@@ -35,7 +36,11 @@ type SettingsRules = {
 }
 
 const settingsRules: SettingsRules = {
-
+  'logo_url' : {
+    description: 'The application logo to display on the a12nserver pages',
+    fromDb: true,
+    default: '/_hal-browser/assets/curveball-logo.svg'
```

Comment:
When I looked at this last night, I *think* this is not the right path for what was released in the curveball browser, so I would unlink your local env and try this with the latest browser package.

Reply:
My bad, was looking at an old version of the browser package.